_posts/2024-03-01-pyos-newsletter-march-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: single
+title: "pyOpenSci Newsletter: March 2024"
+excerpt: "New partnerships, packages, and conference announcements!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-newsletter-march-2024.html
+header:
+  overlay_color: "#666"
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+toc: true
+comments: true
```

Comment:
Just a note i haven't actually tested out the comments system since i set it up which was a few years back 😆 . it might be worth doing so.

Reply:
@lwasser I tested this [on the shark attack post](https://www.pyopensci.org/blog/scipy-2023-community.html) and it looks like comments are still good to go!

Reply:
THANK YOU for testing this. i saw your comment in the discussion which is how it should work. perfect!!